locale/en/LC_MESSAGES/django.po

```diff
@@ -5371,6 +5372,20 @@ msgstr ""
 msgid "SHIBBOLETH_ERROR_USER_ACCOUNT_DISABLED"
 msgstr "The user account has been disabled."
 
+#: templates/403.html
+msgid "ACCESS_RESTRICTED_403"
+msgstr "Access restricted"
+
+#: templates/403.html
+msgid "ACCESS_RESTRICTED_403_INFO"
+msgstr ""
+"You lack the necessary permissions to view this content. If you have a user "
+"ID, remember to log in."
```

Comment:
User ID? Maybe user account is better in this context?

Reply:
Aalto uses the word ``Aalto user ID`` to translate ``Aalto-käyttäjätunnus``.